experiments/2multifsound.py

```diff
@@ -3,112 +3,49 @@
 # write an experiment that creates a new control program.
 import os
 import sys
+import copy
 
 BOREALISPATH = os.environ['BOREALISPATH']
-#sys.path.append(BOREALISPATH + "/experiment_prototype")
+sys.path.append(BOREALISPATH)
 
-#import test
 from experiment_prototype.experiment_prototype import ExperimentPrototype
+import experiments.superdarn_common_fields as scf
 
 class TwoMultifsound(ExperimentPrototype):
 
     def __init__(self):
         cpid = 350300
-        rxrate = 5.0e6
-        output_rx_rate = 10.0e3/3
 
-        tx_ant = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15]
-        rx_main_ant = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15]
-        rx_int_ant = [0, 1, 2, 3]        
-        pulse_sequence = [0, 9, 12, 20, 22, 26, 27] #[0, 14, 22, 24, 27, 31, 42, 43]
-        tau_spacing = 2400 # 1500 # us
         slice_1 = {  # slice_id = 0, the first slice
-            "tx_antennas": tx_ant,
-            "rx_main_antennas": rx_main_ant,
-            "rx_int_antennas": rx_int_ant,
-            "pulse_sequence": pulse_sequence,
-            "pulse_phase_offset": [0, 0, 0, 0, 0, 0, 0],
-            "tau_spacing": tau_spacing,
-            "pulse_len": 300,  # us
-            "num_ranges": 75,  # range gates
-            "first_range": 180,  # first range gate, in km
+            "pulse_sequence": scf.SEQUENCE_7P,
```

Comment:
I guess tx antennas, rx main antennas, rx int antennas and pulse phase offset are in the super?

Reply:
yes